src/main/java/database/DatabaseController.java

```diff
@@ -318,6 +318,12 @@ private LocalDate convertStringDateToLocalDate(String date) {
     private String convertLocalDateToStringDate(LocalDate date) {
         return date.toString();
     }
+
+    //SearchForm
```

Comment:
I don't think we need the search database gateway. It's bascially the same thing as the listingdatabasegateway